onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc

```diff
@@ -292,7 +292,21 @@
         "^test_roialign_cpu",
         "^test_dft_axis_cpu",
         "^test_dft_cpu",
-        "^test_dft_inverse_cpu"
+        "^test_dft_inverse_cpu",
```

Comment:
So we need to add the test to both main.cc and here to disable them? Putting them only in one place doesn't work?

Reply:
Yes

Reply:
@PatriceVignola : Say, if you're ever touching that area of code, I see some that can be re-enabled. e.g.

    broken_tests.insert({"unsqueeze_three_axes", "DML does not support 6D tensors"});
    broken_tests.insert({"unsqueeze_unsorted_axes", "DMLdoes not support 6D tensors"});

(we support 8D now)